azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -105,7 +118,9 @@ func resourceArmApplicationInsights() *schema.Resource {
 
 func resourceArmApplicationInsightsCreateUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*clients.Client).AppInsights.ComponentsClient
+	billingFeatureClient := meta.(*clients.Client).AppInsights.BillingFeatureClient
```

Comment:
```suggestion
	billingClient := meta.(*clients.Client).AppInsights.BillingFeatureClient
```

Reply:
Renamed from billingFeatureClient to billingClient.